### ISSUE TITLE
docker-compose-config.yml sometimes has incorrect BBOX value

### ISSUE THREAD

**Issue by nnhubbard** [openmaptiles/openmaptiles#642]
When I run `./quickstart.sh connecticut` the `docker-compose-config.yml` file that is created contains the following BBOX:

`BBOX: " -74.3018772, 40.9476252, -66.8855449, 47.4598397"`

However, if I look on https://boundingbox.klokantech.com the BBOX should be:

`-73.727775,40.949262,-71.786994,42.050587`

The BBOX that is generated is a MUCH larger area, so much so that it makes the import take over 60 minutes. However, if I correct the BBOX value with what I got from the boundingbox website, it takes only 5 minutes to run the import.

Why is this incorrect value getting created? The `connecticut.osm.pbf metadata contains the correct values, but somehow they are getting changed.

**Comment by eva-j** [comment]
You are right, extent is really much bigger then it should be, when running Connecticut via quickstart.

What kind of metadata did you check please?
http://download.geofabrik.de/north-america/us/connecticut.poly seems to have correct Connecticut boarder, but `osmctools` get from connecticut.osm.pbf different values (that wrong bigger bbox).
I mean if I do the same like download-geofabrik does:
`osmconvert --out-statistics connecticut.osm.pbf`
I get the wrong bbox, so I think the problem is in .osm.pbf file.

It looks like it is not only Connecticut, that get not really good bbox from .osm.pbf. E. g. Switzerland has bbox `5.9559,45.818,10.4921,47.8084`
but osmconvert gets:
```
lon min: 4.5954793
lon max: 11.1844536
lat min: 43.3320379
lat max: 48.0632664
```
I guess there could be some feature starting in the state area but leading quite far away out of it and this way it reads much bigger extent then... This is only a guess, but I tried to clip connecticut.osm.pbf file exactly at the boarder shape using the same tool, I mean:
`osmconvert connecticut.osm.pbf -B=connecticut.poly -o=connecticut-clipped.osm.pbf`
then after asking:
`osmconvert --out-statistics connecticut-clipped.osm.pbf`
I get correct extent:
```
lon min: -73.7279827
lon max: -71.7857740
lat min: 40.9804928
lat max: 42.0521575
```
Maybe we could try to apply this clipping before getting the extent from osm.pbf or just take extent from .poly of the state instead of osm.pbf statistics.

Anyway, you can always put the extent you wish to have in openmaptiles.yaml and .env and make OMT without quickstart.sh.

**Comment by nnhubbard** [comment]
This sounds like a bug. I tested a few more US States and they had the same issue. For example, Washington State gets a much larger BBOX generated and will take 4x the generation time than if I corrected the BBOX.

Are you saying that `openmaptiles` uses the `osmctools` tool to get the BBOX?

**Comment by eva-j** [comment]
Yes, in `download-geofabrik` command (https://github.com/openmaptiles/openmaptiles/blob/master/Makefile#L64) there is `osmconvert` from `osmctools` used to get bbox values:
https://github.com/openmaptiles/import-osm/blob/master/download-geofabrik.sh#L25
